SquareRegisterSDK/SCCAPIResponse.m

```diff
@@ -170,7 +174,7 @@ - (nonnull NSString *)description;
 
 - (NSUInteger)hash;
 {
-    return self.userInfoString.hash ^ self.error.hash ^ self.transactionID.hash;
+    return self.userInfoString.hash ^ self.error.hash ^ self.transactionID.hash ^ self.clientTransactionID.hash;
```

Comment:
Equality comparisons probably should not test against transaction_id anymore, because it won't necessarily be there. client_transaction_id will always be in the response for the latest Register versions.